out.csv

```diff
@@ -0,0 +1,290 @@
+collabid,milestone,visual?,modified,grader,score,comment,,
```

Comment:
If sensitive, remove from *history*. If not, delete. If important, archive in UP data storage.